### PR TITLE
Make producer-app's CronJob schedule optional

### PR DESCRIPTION
**Pull request by torbsto**
In some instances (particularly demos), we don't need a CronJob but a regular Job. This commit makes the schedule optional. If it is not specified, the generated deployment will be of kind 'Job'.

### CODE REVIEW

File: charts/producer-app/templates/job.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {{- $root := . -}}
```
Comment (torbsto) [inline charts/producer-app/templates/job.yaml:1]: I've also renamed this file since `job.yaml` fits both cases. Hope that's okay.

Comment (yannick-roeder) [reply charts/producer-app/templates/job.yaml:1]: Yes, I think this should not cause any problems

**Comment by yannick-roeder** [comment]
LGTM, I will ask @VictorKuenstler to have a look as well

**Review by VictorKuenstler — APPROVED** [review]
LGTM

**Review by yannick-roeder — APPROVED** [review]

**Pull request merged**